packages/core/src/browser/frontend-application-module.ts

```diff
@@ -188,8 +188,10 @@ export const frontendApplicationModule = new ContainerModule((bind, unbind, isBo
     bind(LabelProviderContribution).to(DefaultUriLabelProviderContribution).inSingletonScope();
     bind(LabelProviderContribution).to(DiffUriLabelProviderContribution).inSingletonScope();
 
-    bind(PreferenceProvider).toSelf().inSingletonScope().whenTargetNamed(PreferenceScope.User);
-    bind(PreferenceProvider).toSelf().inSingletonScope().whenTargetNamed(PreferenceScope.Workspace);
+    bindDefaultPreferenceProvider(bind);
+    bind(PreferenceProvider).toSelf().inSingletonScope().whenTargetNamed(PreferenceScope.USER);
```

Comment:
renaming enum literals is breaking change, are you sure there are not extension relying on it. We agreed to not break if we are not sure or go through deprecation: https://github.com/theia-ide/theia/wiki/Dev-Meetings#api-deprecation